src/com/lftechnology/java/training/dipak/employeemanagement/dao/LoginDao.java

```diff
@@ -0,0 +1,95 @@
+
+package com.lftechnology.java.training.dipak.employeemanagement.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import com.lftechnology.java.training.dipak.employeemanagement.api.UserApi;
+import com.lftechnology.java.training.dipak.employeemanagement.domain.Employee;
+import com.lftechnology.java.training.dipak.employeemanagement.domain.User;
+import com.lftechnology.java.training.dipak.employeemanagement.domain.UserType;
+
+/**
+ * <p>This class contains method that interact with database to check whether
+ * the username and password entered is valid or not.</p>
+ * 
+ * @author Dipak Thapa<dipakthapa@lftechnology.com>
+ */
+public class LoginDao implements UserApi {
+	private final String  CONNECTION_FAILED_MESSAGE="Connection Setup Failed";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this field "CONNECTION_FAILED_MESSAGE" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00116)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this final field static too. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1170)